primefaces/src/main/resources/META-INF/resources/primefaces/datatable/datatable.frozen.js

```diff
@@ -875,5 +887,56 @@ PrimeFaces.widget.FrozenDataTable = PrimeFaces.widget.DataTable.extend({
 
         scrollRows.children('td.ui-duplicated-column').remove();
         frozenRows.children('td.ui-duplicated-column').remove();
+    },
+
+    /**
+     * Adjusts the height of the body and foot rows to fit the current settings.
+     */
+    fixRowHeights: function() {
+        this._fixRowHeights(this.scrollThead.children(), this.frozenThead.children());
+        this._fixRowHeights(this.scrollTbody.children(), this.frozenTbody.children());
+        var frozenFootRows = this.frozenTfoot.children();
+        if (frozenFootRows.length > 0) {
+            this._fixRowHeights(this.scrollTfoot.children(), frozenFootRows);
+            var scrollBarHeight = this.scrollContainer.height() - this.frozenContainer.height();
+            if (scrollBarHeight > 0) {
+                var browser = PrimeFaces.env.browser;
+                if (browser.webkit === true || browser.mozilla === true) {
+                    this.frozenBody.append('<div style="height:' + scrollBarHeight + 'px"></div>');
+                } else {
+                    this.frozenBodyTable.css('margin-bottom', scrollBarHeight);
+                }
+            }
+        }
+    },
+
+    /**
+     * Adjusts the height of the given rows to fit the current settings.
+     * @protected
+     * @param {JQuery} scrollRows The scrollable rows to adjust.
+     * @param {JQuery} frozenRows The frozen rows to adjust.
+     */
+    _fixRowHeights: function(scrollRows, frozenRows) {
```

Comment:
Can't we rename the previous function to something like `fixRowHeightsAll` and remove the underscore from this one?

Reply:
@jepsar you can make changes right to this PR if you want to.

Reply:
Renamed the functions.